exasol-s3/pom.xml

```diff
@@ -33,17 +34,15 @@
             <groupId>org.apache.hadoop</groupId>
             <artifactId>hadoop-client</artifactId>
             <scope>provided</scope>
-            <exclusions>
-                <exclusion>
-                    <groupId>javax.ws.rs</groupId>
-                    <artifactId>*</artifactId>
-                </exclusion>
-            </exclusions>
+        </dependency>
+        <dependency>
+            <groupId>io.netty</groupId>
+            <artifactId>netty-all</artifactId>
```

Comment:
Do you really need `netty-all`? Isn't there a subset that would suffice?

Reply:
Yes, this is mostly to fix the vulnerabilities, the ones required have CVEs. Adding this uses the up-to-date versions.

But good point, this should be with `provided` scope.